CHANGELOG.md

```diff
@@ -0,0 +1,134 @@
+- The `overflow-x` and `overflow-y` mixins no longer set the max width and height respectively. If you were making direct use of the mixins, you’ll need to add max widths or heights to your CSS, as needed.
+
+# [1.0.3] - 2021-03-15
+
+## Added
+
+- `.o-notification-center` component, that handles layout of notifications in the top right of the viewport, overlaying other content
+- `.a-flash` component to be used for displaying global updates at the top of the viewport, or at the top of a form, in the document flow
+- Negative margin utility classes `.u-margin-neg-#{$size}`
+
+## Changed
+
+- Most of our button types now only alter the base button’s `border-color`, not the shorthand `border`, to ensure equal heights of buttons
+- Input elements now only alter the base style’s `border-color`, not the shorthand `border`, to ensure equal heights and consistent appearance
+- Simplifies the way we import webfonts, and allows easier setup and overriding, using a more explicit `$bitstyles-webfont-variants` nested map. Filenames no longer need to match the previous format, though it’s advised
+
+# [1.0.2] - 2021-03-11
+
+## Added
+
+- `.u-border` utility classes
+- Storybok documentation now includes the `a11y-addon` for in-browser checking of some a11y low-hanging fruit
+- Select elements can now display different border-color, background-color etc when they are invalid. There are examples showing how to correctly markup invalid selects (it is quite specific due to browser implementations)
+- There’s a new section-heading component in the documentation, with variants containing buttons
+- Flex and box-alignment classes are now available at various breakpoints
+
+## Changed
+
+- `.a-button--tab` indicator color is now set by a specific `$bitstyles-button-tab-border-color` sass variable instead of using the border variable. Borders can now be set on `a-button--tab`s, whereas before they could not (by default they’re all transparent, so there will be no visual change)
+- Webfonts are now imported using a base URL and several variants specified in $bitstyles-webfont-variants. The filenames of the webfont files need to match the convention `${base-url}-${variant}.${woff|woff2}`
+- As per the current state of AT interpretation of forms, examples of select element usage now place the label as sibling to (not parent of) the select. If you were using the previous structure, please switch
+
+# [1.0.1] - 2021-03-04
+
+## Added
+
+- `.u-round` for applying rounded corners to elements
+- `.o-ui-group` for visually binding several closely-related interactive elements (buttons or inputs)
+- `.a-button--ui`, for buttons that are not the primary action of a screen
+- `.a-card` is now available at the `m` breakpoint
+- Example form layouts for login & signup flows
+- `.u-grid` and variants are now available at `l` breakpoint
+- `.u-padding` and variants are now available at various breakpoints
+
+## Changed
+
+- Uses github actions to deploy our storybook to github pages
```

Comment:
I think such internal things I would not mention. After all a changelog should only contain user-relevant information imho :)